### ISSUE TITLE
Support Bootstrap 5 alpha 3 floating labels

### ISSUE THREAD

**Issue by thimo** [bootstrap-ruby/bootstrap_form#573]
Bootstrap 5 alpha 3 supports floating labels: https://v5.getbootstrap.com/docs/5.0/forms/floating-labels/

It would be nice to enable this in bootstrap_form with an option, something like `<%= f.text_field :query, floating: true %>`.

To enable this a few things must be done in the rendering:

1. use `form-floating` as the control class
2. revert order of `input` and `label` elements
3. add `placeholder` to input field, fill with label value by default

```
<div class="form-floating mb-3">
  <input placeholder="Search" class="form-control" type="text" name="filter[query]" id="filter_query">
  <label for="filter_query">Search</label>
</div>
```

I would love to pick this up, but if anybody else has the time, please feel free to do so. Would also love feedback on how to enable this functionality, is `floating: true` the right way to do that?

**Comment by lcreid** [comment]
Thanks for proposing this. We definitely want to support floating labels. (We want to support whatever Bootstrap supports.)

We'd be very interested in seeing a PR if you want to take a crack at it. When we see how it works I think we can have better feedback for you. Feel free to ask questions here if you're wondering about existing code.

Thanks for your interest and input in `bootstrap_form`.

**Comment by lcreid** [comment]
Closed by #584 .
